libs/novui/src/recipes/input.recipe.ts

```diff
@@ -97,6 +99,30 @@ export const INPUT_RECIPE = defineSlotRecipe({
     required: {
       color: 'typography.text.feedback.required !important',
     },
-    wrapper: {},
+    wrapper: {
+      // prevent Mantine from injecting 2px margin that causes layout shift
+      marginBottom: '0 !important',
+    },
+  },
+  variants: {
+    variant: {
+      // TODO: determine if we want this built-in! Prevents layout shift with error states
+      preventLayoutShift: {
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This addresses a UX concern with Inputs shifting when an error message appears / disappears. However, it requires saving space for the error to accomplish this, so it adds some additional white space below each input if this variant is used. I chose to do this as a variant for the time being because it's "opt-in" so we can pick and choose for the time being where we're most concerned about the layout shift‏